projects/plutus/src/main.c

```diff
@@ -1,67 +1,58 @@
 #include <stddef.h>
 
 #include "delay.h"
-#include "event_queue.h"
-#include "gpio.h"
-#include "interrupt.h"
 #include "log.h"
-#include "ltc_afe.h"
 #include "plutus_cfg.h"
-#include "soft_timer.h"
-#include "spi.h"
+
+#include "can_transmit.h"
+#include "fault_monitor.h"
+#include "plutus_sys.h"
 #include "wait.h"
 
-static LtcAfeStorage s_afe;
+static PlutusSysStorage s_plutus;
+static FaultMonitorStorage s_fault_monitor;
 
-static void prv_cell_conv_cb(uint16_t *result_arr, size_t len, void *context) {
-  // disabled until CAN ACK timeout is fixed
-  // for (size_t i = 0; i < len; i++) {
-  //   LOG_DEBUG("C%d: %d.%dmV\n", i, result_arr[i] / 10, result_arr[i] % 10);
-  // }
+static size_t i = 0;
```

Comment:
`s_i`? Also this is a bad name maybe `s_cell_index`

Reply:
Yeah, I need to figure out why we're dropping CAN messages. This is intended as a temporary solution.